design/API/NT-API.yml

```
@@ -2892,9 +2894,237 @@ paths:
                       knowledge:
                         type: array
                         items:
-                          $ref: "#/components/schemas/Knowledge"
+                          $ref: "#/components/schemas/Knowledges"
                       page:
-                        $ref: "#/components/schemas/Page"
+                        $ref: '#/components/schemas/Page'
+    patch:
+      summary: Update Knowledge Item
+      tags: []
+      operationId: updateKnowledge  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)